.github/workflows/checks.yml

```diff
@@ -110,8 +110,19 @@ jobs:
       - name: Test
         run: npm run test --workspaces
 
+  size:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Size check
+        uses: andresz1/size-limit-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I like this approach to implementing a limit.  It doesn't enforce a hard limit but it makes changes visible, so a very useful PR isn't blocked by increasing the size slightly, but it does focus attention on the subject.  A similar solution might work with CI runtime.  Rather than rejecting a PR that increases runtime, e.g. because it adds more tests, it could effectively ask the author to justify any increase by highlighting the delta.

Reply:
Actually PR will be rejected if limit is reached.

Just tested and got a bubbled exception.

> Error: Size limit has been exceeded.

From GitHub action documentation:

> Commenting pull requests with the comparison of Size Limit output.
> Rejecting a pull request if the cost exceeds the limit.

That said, we can always adapt or remove the limit in `package.json` if it would become an issue. So no one should be blocked because of it.

Side note: same strategy as in agent-js.

Thanks for the review